src/test/java/redis/clients/jedis/commands/jedis/BitCommandsTest.java

```diff
@@ -182,6 +191,15 @@ public void bitCount() {
 
     assertEquals(3, (long) jedis.bitcount("foo", 2L, 5L));
     assertEquals(3, (long) jedis.bitcount("foo".getBytes(), 2L, 5L));
+  }
+
+  @Test
+  @SinceRedisVersion("7.0.0")
+  public void bitCountByteOptions() {
```

Comment:
BYTE is actually the default mode and BIT is the new option. Modifier is safer name and aligns with other tests.

```suggestion
  public void bitCountModifier() {
```